docs/guide/responses.md

```diff
@@ -93,6 +93,7 @@ inertia_config(
     "Storefront/#{path.camelize}/#{action.camelize}"
   end
 )
+````
```

Comment:
This code block has 4 backticks opening and closing

````markdown
```ruby
inertia_config(
  component_path_resolver: ->(path:, action:) do
    "Storefront/#{path.camelize}/#{action.camelize}"
  end
)
```
````